masarykova-univerzita-pravnicka-fakulta.csl

```diff
@@ -238,27 +238,33 @@
   <macro name="issued">
     <choose>
       <if type="article-journal article-magazine article-newspaper" match="any">
-        <choose>
-          <if variable="issued" match="any">
-            <date variable="issued">
-              <date-part name="year" range-delimiter="&#8211;"/>
-            </date>
-          </if>
-          <else>
+        <group delimiter=", ">
```

Comment:
You don't actually have to test for a variable before printing it: the test is implicit, i.e.:
```
          <group>
            <choose>
              <if variable="issued" match="any">
                <date variable="issued">
                  <date-part name="year" range-delimiter="&#8211;"/>
                </date>
              </if>
            </choose>
          </group>
```

is 100% identical to

```
                <date variable="issued">
                  <date-part name="year" range-delimiter="&#8211;"/>
                </date>
```